component/api/src/main/java/org/exoplatform/social/core/space/SpaceListAccess.java

```diff
@@ -31,12 +31,18 @@
   
   /** The user id. */
   private String userId;
+
+  /** The visited profile id, used while getting list of common spaces between two users. */
+  private String profileUserId;
```

Comment:
I find this parameter name not relevant. Can we find something else ?

It could be 
```suggestion
  private String otherUserId;
```

Reply:
agree